app/script/conversation/EventMapper.coffee

```diff
@@ -130,6 +147,7 @@ class z.conversation.EventMapper
   _map_all_verified: ->
     message_et = new z.entity.Message()
```

Comment:
We could probably use fitting message entity for this as well and handle all e2ee related messages in a single entity type. z.entity.DegradedMessage -> z.entity.E2EMessage and degraded_message_type -> 'e2ee_message_type'